### DISCUSSION TITLE
The game is still punishing unlucky players

### DISCUSSION THREAD

**Discussion by MrPunish33r** [sunflower-land/sunflower-land#2327] [General]
So, We "unlucky players" have been at a disadvantage of resources for a long time now. but it wasn't a big deal until now.
Now when the new season has been released and the race to the cabbage girl began, the bad design of the event favors people with 40 plots and more, while it heavily punishes people with 39 and less.
Devs surely know a lot of people are upset with the unbalance, and are thinking of adding more balance to the game, but why on earth did you not think about balancing the event first ? you say from now on the game will reward active players, but no matter how much active I am or will be, I will never be able to compete with people who have 40 plots. 
the quests I'm referring to are: harvest 20 radishes, 20 wheat, 20 kale and god knows how many other quests like this are coming along the way, people with 40 plots can do 20 Radishes and 20 wheat in one go, while who have less will have to add another cycle of at least 17 hours to just harvest 1 more !!!!!!! imagine what the difference will be when quests like this stack up. it may be days not just hours. 
it's really a bad though out event, and I think should've been taken into consideration from the beginning if you're really want to favor gameplay and hardworking people, and not just "luck". The game is really becoming frustrating to play

**Comment by offerT** [comment]
Agree， the numbers of 80 and 20 is very disturbing for me, because I only have 38 and too many people have 40+.
There is rng in resources, but they should not continue to be troubled by tasks

**Comment by ghost** [comment]
The problem with quests is not that people do not have an equal amount of resources, but that there is no balance of tasks. It is necessary to give the right to choose a quest, which will allow players to choose what they can do, and also such a system will avoid early preparation for the task.
